media/css/firefox/browsers-products.scss

```diff
@@ -570,71 +570,41 @@ $url-download-link-hover: svg-url($download-link-hover);
     }
 }
 
-.fxa-email-form-title {
-    @include font-size(24px);
-    margin-bottom: $spacing-xl;
-}
-
 .fxa-email-form-intro {
     display: none;
 }
 
-.fxa-email-field-container {
-    .field {
-        label {
-            @include text-body-sm;
-            color: $color-white;
-        }
-
-        input[type="email"] {
-            box-shadow: none;
-            padding: 12px 16px;
-            width: 100%;
-        }
-    }
-
-    .mzp-c-button {
-        width: 100%;
-    }
+.mzp-c-field-info {
+    @include text-body-xs;
+}
 
+.fxa-email-field-container {
     @media #{$mq-md} {
         @supports(display:grid) {
             & {
                 @include grid-column-gap($spacing-lg);
                 display: grid;
-                grid-template-columns: 1fr min-content;
-                grid-template-rows: auto auto auto;
+                grid-template-columns: 1fr 100px;
             }
 
-            .field {
-                grid-column-start: 1;
-                grid-row-start: 1;
-                grid-row-end: 3;
-                margin-bottom: 0;
-
-                label {
-                    margin-bottom: $spacing-sm;
-                }
+            .mzp-c-field {
+                padding-bottom: 0;
             }
 
-            .agreement {
-                grid-column-start: 1;
-                grid-row-start: 3;
-                grid-row-end: 4;
-                margin-top: $spacing-sm;
-                margin-bottom: 0;
-            }
+            .mzp-c-button-container {
+                display: block; // not flex
 
-            .mzp-c-button {
-                grid-column-start: 2;
-                grid-row-start: 2;
-                grid-row-end: 3;
+                // put in fake form label to push button down height of real form label next to it
```

Comment:
Curious, does this go further by addressing something that some extra `padding-top` couldn't fix?

Reply:
It will reflect the text size, including the user's personal settings, more accurately than a px or em padding value.